### PR TITLE
Remove unused code in tools/wpt/utils.py

### PR DESCRIPTION
**Pull request by foolip**
These were determined to be unused by grepping for the names and looking
at all matches. Note

### CODE REVIEW

**Comment by gsnedders** [comment]
The CI failure tends to suggest they aren't all unused. :)

It appears at least `seekable` is used.

**Comment by gsnedders** [comment]
The CI failure tends to suggest they aren't all unused. :)

It appears at least `seekable` is used.

File: tools/wpt/utils.py [hunk 1]
```diff
@@ -49,30 +49,6 @@ def call(*args):
...
-            logger.debug(" ".join(full_cmd))
-            return subprocess.check_output(full_cmd, cwd=repo_path, stderr=subprocess.STDOUT).strip()
-        except subprocess.CalledProcessError as e:
-            logger.error("Git command exited with status %i" % e.returncode)
-            logger.error(e.output)
-            sys.exit(1)
-    return git
-
-
-def seekable(fileobj):
```
[the remark this answers is not in the document]
Comment (foolip) [reply tools/wpt/utils.py]: I don't understand how I failed at grepping this :(

**Comment by gsnedders** [comment]
```
./utils.py:4:1: F401 'sys' imported but unused
```

**Review by jgraham — APPROVED** [review]

**Pull request merged**